cmd/machine-controller/main.go

```diff
@@ -169,7 +169,7 @@ func main() {
 	flag.BoolVar(&nodeCSRApprover, "node-csr-approver", true, "Enable NodeCSRApprover controller to automatically approve node serving certificate requests")
 	flag.StringVar(&podCidr, "pod-cidr", "172.25.0.0/16", "The network ranges from which POD networks are allocated")
 	flag.StringVar(&nodePortRange, "node-port-range", "30000-32767", "A port range to reserve for services with NodePort visibility")
-
+	flag.StringVar(&nodeRegistryCredentialsSecret, "node-registry-credentials-secret", "", "A Secret object reference, that containt auth info for image registry in namespace/secret-name form, example: kube-system/registry-credentials. Secret format: map[string]github.com/containerd/containerd/pkg/cri/config.AuthConfig")
```

Comment:
This isn't intuitive enough and seems a bit off to me 
```
Secret format: map[string]github.com/containerd/containerd/pkg/cri/config.AuthConfig
```

Maybe it can be replaced with something like 
```
Secret format: map[string]AuthConfig with AuthConfig defined at https://github.com/kubermatic/machine-controller/blob/master/pkg/containerruntime/containerd.go#L223
```

Reply:
Just a nit, if you think it's clear enough then let's just go with it

Reply:
Added link to the doc where format and structure is explained